service/test/fixtures/d-installer-archs.yaml

```diff
@@ -0,0 +1,291 @@
+products:
```

Comment:
Do we need such a comprehensive configuration file just for testing archs? I looses focus and can tempt others to (ab)use the same fixture for other unrelated tests (like storage volumes). I would include the bare minimal content that relates to the test.

Reply:
well, I try to be consistent with test for single product which is also quite big - https://github.com/yast/d-installer/blob/master/service/test/fixtures/d-installer-single.yaml
I will be more then happy to reduce size of fixture.

Reply:
You can reduce the size, especially the storage one, but remember that all the keys should be included AFAIK.

Reply:
ok, I will try to reduce it as much as possible.